snuba/util.py

```diff
@@ -95,6 +96,16 @@ def function_expr(fn: str, args_expr: str='') -> str:
     to more complex expressions.
 
     """
+    if fn.startswith('apdex('):
+        match = APDEX_FUNCTION_RE.match(fn)
```

Comment:
to be clear, this approach means snuba is not going to process and resolve the parameters passed to `apdex`.
This can work as a short term solution but it will prevent any other column processing on the parameters.
Example:
We rewrite columns often (not the case of duration though), this will not happen here since Snuba does not know that parameter is a column and will consider them literals.
In order to have the parameters to apdex processed, you need to read them from the args_expr argument, and then this would have to be used in selected_columns and groupby ignoring the aggregations field.

Reply:
that might be acceptable until we have a good way to handle multi arg functions?

Reply:
I am ok with that, we just need to keep in mind this limitation